src/tests/ftest/io/large_file_count.yaml

```diff
@@ -1,6 +1,6 @@
 hosts:
-  test_servers: 3
-  test_clients: 1
+  test_servers: 5
+  test_clients: 3
```

Comment:
Have we considered 2 servers, 2 clients so it still fits on Medium?

Reply:
This test never has fit on medium, it's failed ever since it was moved there.  I think this is the correct way forward.